extensions/amp-consent/integrating-consent.md

```diff
@@ -40,13 +40,19 @@ window.parent.postMessage(
   {
     type: 'consent-ui',
     action: 'ready',
+    initialHeight: (optional string, default `30vh`),
+    enableBoarder: (optional boolean, default true),
```

Comment:
There's a typo in `enableBorder` here and also on line 54

```suggestion
    enableBorder: (optional boolean, default true),
```

Reply:
Thanks!